speed_expr/speed_runner.cpp

```diff
@@ -0,0 +1,27 @@
+#include <string>
```

Comment:
What exactly does this file do?

Reply:
Oh I see.

Reply:
So this runs a single workload. I think we should rename this file to make it a little more clear. Maybe something like stream_process_speed.cpp?